addons/fools/addon.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "eastah egg",
+  "description": "o_0",
+  "tags": ["community"],
```

This is probably not neccasary to do at all, but
```suggestion
  "tags": ["community", "new"],
```

---

The new tag is automatically added based on `versionAdded` and [latestUpdate](https://scratchaddons.com/docs/reference/addon-manifest/#latestupdate).

---

> The new tag is automatically added based on `versionAdded` and [latestUpdate](https://scratchaddons.com/docs/reference/addon-manifest/#latestupdate).

Ah, my bad then.